newsroom/agenda/formatters/csv_formatter.py

```diff
@@ -43,7 +43,7 @@ def format_event(self, item: Dict[str, Any]) -> Dict[str, Any]:
         event = item.get("event", {})
         return {
             "Event name": item.get("name", ""),
-            "Description": item.get("definition_long", item.get("definition_short", "")),
+            "Description": item.get("definition_long") or item.get("definition_short", ""),
```

Comment:
would do it probably `item.get("definition_long") or item.get("definition_short") or ""` so we make sure it's always string

Reply:
yep